solidity/package.json

```
@@ -18,9 +18,14 @@
     "test:quick": "mocha --exit --recursive --timeout 45000",
     "test:quick:watch": "mocha --watch --exit --recursive --timeout 45000",
     "demo": "truffle migrate --reset && truffle exec ./scripts/delegate-tokens.js",
-    "lint": "solium --dir ./contracts",
+    "lint": "npm run lint:js && npm run lint:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
```

Maybe we could simplify the configuration by removing `config.eslintPaths` files in all folders exept `dashboard`?
```suggestion
    "lint:js": "eslint --ignore-pattern 'dashboard/' .",
```

---

Not sure if we can, it didn't work when I tried. According to the [docs](https://eslint.org/docs/user-guide/command-line-interface) we need to provide file or dir to run `eslint`. Did you try and it work for you?

---

We can do `eslint .` and it will check recursively all the files. If we want to skip some directories or files we can define them in a `.eslintignore` file or pass with `--ignore-pattern` as in the example above.

---

'2a87ff5' Adding a dot right after eslint `eslint .` did the trick!